src/hydro_forces.cpp

```diff
@@ -353,58 +353,48 @@ double TestHydro::setVelHistory(double val, int step, int b_num, int index) {
 *******************************************************************************/
 std::vector<double> TestHydro::ComputeForceHydrostatics() {
 	std::vector<double> position, displacement;
```

Comment:
If we want to remove std::vector<double>s, then maybe change this to Eigen3 types (or their Chrono counterparts)

Reply:
Or just remove this line, since these are redefined this way farther down (line 368,369,371 in changed file)

Reply:
Removed it, it is not used in the new implementation